app/controllers/admin/application_controller.rb

```diff
@@ -27,6 +28,23 @@ def authorize_instance_access
     authorize! :manage, requested_resource
   end
 
+  def set_time_zone
+    begin
+      logger.info 'I am the before action'
```

Comment:
Try not to commit logging messages unless they're useful long-term. Can we pull these out or log the timezone change more informatively?